### ISSUE TITLE
riak-cs rpm doesn't properly install on Fedora 18

### ISSUE THREAD

**Issue by stahnma** [basho/riak_cs#554]
I grabbed the F17 rpm thinking it would likely work on F18. It doesn't. I realize that's not supported, but you might want to fix it up. It just has to do with filesystem ownership.

<pre>
[root@localhost ~]# rpm -Uvh http://s3.amazonaws.com/downloads.basho.com/riak-cs/1.3/1.3.1/fedora/17/riak-cs-1.3.1-1.fc17.x86_64.rpm
Retrieving http://s3.amazonaws.com/downloads.basho.com/riak-cs/1.3/1.3.1/fedora/17/riak-cs-1.3.1-1.fc17.x86_64.rpm
Preparing...                          ################################# [100%]
    file /usr/sbin from install of riak-cs-1.3.1-1.fc17.x86_64 conflicts with file from package filesystem-3.1-2.fc18.x86_64
    file /usr/share/man/man1 from install of riak-cs-1.3.1-1.fc17.x86_64 conflicts with file from package filesystem-3.1-2.fc18.x86_64
</pre>

Basically a better %files section in the spec file without greedily grabbing /usr/sbin et al would solve that issue.

This behavior from RPM was new in F18, IIRC.

**Comment by jaredmorrow** [comment]
Thanks for reporting this @stahnma I'll take a look

**Comment by jaredmorrow** [comment]
Fixed in node_package, will be fixed in next release of stanchion/riak_cs.
